tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+cookiecutter==0.9.0
+coverage==3.7.1
+ddt==0.8.1
+diff-cover==0.7.2
+Django==1.4.16
+django_nose==1.2
+fs==0.5.0
+lazy==1.2
+mock==1.0.1
+nose==1.3.4
+nose-parameterized==0.3.5
+pep8==1.5.7
+pylint==0.28
+pypng==0.0.17
+rednose==0.4.1
+requests==2.4.3
+selenium==2.44.0
+simplejson==3.6.5
+unicodecsv==0.9.4
+webob==1.4
```

Comment:
I removed all transitive dependencies, except XBlock. The main pain points when trying to run the tests from a pull request that somebody put up were really the correct version of xblock-sdk and the correct version of XBlock ("oh, make sure to use my fork of xblock-sdk, and make sure to include this patch when you install XBlock..."). The other dependencies probably aren't going to cause much trouble.